workflows/sarek.nf

```diff
@@ -48,12 +48,18 @@ def checkPathParamList = [
     params.known_snps_tbi,
     params.mappability,
     params.multiqc_config,
+    params.ngscheckmate_bed,
     params.pon,
     params.pon_tbi,
+    params.sentieon_dnascope_model,
+    params.snpeff_cache,
     params.spliceai_indel,
     params.spliceai_indel_tbi,
     params.spliceai_snv,
-    params.spliceai_snv_tbi
+    params.spliceai_snv_tbi,
+    params.bcftools_annotations,
+    params.bcftools_annotations_index,
+    params.bcftools_header_lines
```

Comment:
why?

Reply:
My question was more why not sorted actually

Reply:
should've known

Reply:
fixed